### PR TITLE
added  CLAP_EXT_AUDIO_PORTS_CONFIG_INFO extension

### PR DESCRIPTION
**Pull request by defiantnerd**
added possibility to inquiry the exact bus layout for an configuration.

### CODE REVIEW

**Comment by abique** [comment]
You need to update the surround and ambisonic extensions too.

**Comment by abique** [comment]
Please could you also add the changelog entry?

File: include/clap/ext/audio-ports-config.h [hunk 1]
```diff
@@ -72,6 +72,13 @@ typedef struct clap_plugin_audio_ports_config {
 
 typedef struct clap_plugin_audio_ports_config_info {
   // get info about about an audio port.
+
+  // gets the clap_id of the currently selected config
+  // clap_id might now match the index, can be used to call
+  // clap_plugin_audio_ports_config::select()
+  // [main-thread]
+  clap_id(CLAP_ABI* current_config)(const clap_plugin_t* plugin);
```
Comment (robbert-vdh) [inline include/clap/ext/audio-ports-config.h:80]: Is this needed? The host is the one selecting a config. So it knows the current config.

Comment (defiantnerd) [reply include/clap/ext/audio-ports-config.h:80]: not necessarily - the plugin itself could also switch configurations (like in a user selection, as the commentary states) and the host would not know which one is selected now.

**Pull request merged**